examples/dex/contracts/RecurringPayment.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@acala-network/contracts/schedule/ISchedule.sol";
+import "@acala-network/contracts/utils/Address.sol";
+
+contract RecurringPayment is ADDRESS {
+    uint period;
+    uint remainingCount;
+    uint amount;
+    address payable to;
+    bool private initialized;
+    ISchedule scheduler = ISchedule(ADDRESS.Schedule);
+
+    constructor(uint _period, uint _count, uint _amount, address payable _to) public {
```

Comment:
The command `rush build` gives a warning here: 

```
Warning: Visibility for constructor is ignored. If you want the contract to be non-deployable, making it "abstract" is sufficient.
  --> contracts/RecurringPayment.sol:15:5:
   |
15 |     constructor(uint _period, uint _count, uint _amount, address payable _to) public {
   |     ^ (Relevant source part starts here and spans across multiple lines).

```

Reply:
yeah I also see that warning, not sure if it will affect the functionality...

Reply:
I have tested them and now all the example tests are OK